examples/chip-tool/main.cpp

```diff
@@ -100,7 +100,7 @@ static bool ContentMayBeADataModelMessage(System::PacketBuffer * buffer)
 static void HandleDataModelMessage(System::PacketBuffer * buffer)
 {
     EmberApsFrame frame;
-    bool ok = extractApsFrame(buffer->Start(), buffer->DataLength(), &frame);
+    bool ok = extractApsFrame(buffer->Start(), buffer->DataLength(), &frame) > 0 ? true : false;
```

Comment:
nit: the pattern `foo ? true : false` seems a bit verbose to me and would use `foo` instead. How about:

```
bool ok = (extractApsFrame(buffer->Start(), buffer->DataLength(), &frame) > 0);
```

here and below?

or maybe consde that ok is only used within the if statement and include it as:

```
if (extractApsFrame(buffer->Start(), buffer->DataLength(), &frame) == 0) {
  printf("APS frame processing failure!\n");
  System::PacketBuffer::Free(buffer);
  return;
}

printf("APS frame processing success!\n");
```

although that starts to seem like a larger change, so mabe can be done in a separate PR.